operate/client/src/modules/utils/filter/v2/getElementInstanceStateFilter.test.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {getElementInstanceStateFilter} from './getElementInstanceStateFilter';
+import type {BusinessObjects} from 'bpmn-js/lib/NavigatedViewer';
+
+describe('getElementInstanceStateFilter', () => {
+  it('returns undefined when no element is selected', () => {
```

Comment:
❓I noticed that a lot of test names in the codebase start with `it('should ....')` and followed it as well when adding new tests.

Personally I prefer to [skip the fill word and start with a verb](https://www.epicweb.dev/talks/how-to-write-better-test-names) as well, just like you did in your tests. Did we "officially" change the convention for newer tests? Do we even have a strong convention?